recipes/extract_model/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "extract_model" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/axiom-data-science/extract_model/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5257438da53c8f568f6d21eab798ea8725fae905d78c84b6870fc0ee84ab1961
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python ==3.8
+    - pip
+    - setuptools-scm
+  run:
+    - cf_xarray
+    - cmocean
+    - dask
+    - esmf =8.1.1=mpi_openmpi_h852a670_0  # [linux64]
+    - esmpy =8.1.1=mpi_openmpi_py38h9ab444c_0  # [linux64]
+    - esmf =8.1.1=mpi_openmpi_h047fafc_0  # [osx]
+    - esmpy =8.1.1=mpi_openmpi_py38hf504846_0  # [osx]
```

Comment:
Can you explain what you’re trying to do with these four lines? Is it just about limiting the version?

Reply:
I already fixed that in a PR to @kthyng branch @chrisburr. It should work as soon as they merge it upstream,